### DISCUSSION TITLE
When will the mixtral branch be merged to master?

### DISCUSSION THREAD

**Discussion by AaronYFQ** [ggml-org/llama.cpp#4779] [Q&A] [answered]
Currently mixtral8x7B is a fantastic model, which is supported on mixtral branch. But it's not supported on the related bindings, such as llama-cpp-python. Once it's merged on master,  it shall be supported quickly by the related bindings. great thanks to everyone contributing this project.

**Comment by cryolite-ai** [comment, marked as answer]
@AaronYFQ I think it's merged now on master - I don't know if it's been propagated downstream into other projects, but I think you can now close this here as answered and take things up with the downstream repos ;-)

Joking aside - I'm worried about the number of issues (808 at time of writing), discussions (30+ screens worth) etc which the repo has - it looks like the project is getting a bit swamped with open threads and that can have an impact on morale as well as productivity - namely getting PRs reviewed over the line and merged. If you could close this off asap it would be step in the right direction. Many thanks.
